orc/src/test/java/org/apache/iceberg/orc/TestTableProperties.java

```diff
@@ -50,7 +49,8 @@ public class TestTableProperties {
           Types.NestedField.optional(1, "id", Types.IntegerType.get()),
           Types.NestedField.optional(2, "data", Types.StringType.get()));
 
-  @ClassRule public static final TemporaryFolder TEMPORARY_FOLDER = new TemporaryFolder();
+  @TempDir File folder;
+  @TempDir File testFile;
```

Comment:
both can be private

Reply:
I've changed 'folder','testFile' to private